docs/tutorials/buttons.md

```diff
@@ -0,0 +1,115 @@
+---
+title: Using the Buttons
+subject: Buttons
+---
+
+* Table of Contents
+{:toc}
+
+![EV3 Buttons](/images/lego_ev3_buttons.jpg)
```

Comment:
I didn't ask, it is from a robotics school in japan:
http://www.afrel.co.jp/en/archives/850

I can chop it if you want